packages/react-native-sdk/src/components/call/CallControls/HangupCallButton.tsx

```diff
@@ -16,15 +16,24 @@ export type HangUpCallButtonProps = {
    * @returns void
    */
   onPressHandler?: () => void;
-
+  /**
+   * Handler to be called when the call is hanged up.
+   */
+  onHangUpCall?: () => void;
```

Comment:
I see the usecase but its confusing with two handlers..

I don't think other SDKs have it, can you please check and align

Reply:
React has it named as `onLeave`

Reply:
if it makes it easy to have it, then we can add a clear comment that if onPress is passed the other handler is not invoked

Reply:
Done ✅